test/feature/support/helpers.coffee

```diff
@@ -14,18 +14,19 @@ class Helpers
 
 
   # Creates a new exprestive app in @appPath
-  createExprestiveApp: (optionsStr, done) ->
+  createExprestiveApp: ({library, exprestiveOptions}, done) ->
     serverContents = """
       # Initialize exprestive application
-      express = require 'express'
+      #{library} = require '#{library}'
```

Comment:
And why do you put the `library` argument in a string here? Is it sometimes not a string?